### DISCUSSION TITLE
CSS Reset: position relative as default.

### DISCUSSION THREAD

**Discussion by luigiminardim** [chakra-ui/chakra-ui#3882] [Ideas]
In my experience, it has been a little bit hard to absolute position components on the screen since I have to set the __position__ attribute as __absolute__ in the element and __position relative__ to its parent. I think that chakra-ui could be a little bit more opinionative about what should be the standards of HTML and CSS. Furthermore, I propose that position relative should be as default in _CSSReset_. What does the community think about that?

**Comment by ivan-kleshnin** [comment]
Do you propose to add `position: relative` to every element Chakra renders?! Sounds like a very bad idea to me.
